spec/support/stub.rb

```diff
@@ -0,0 +1,9 @@
+module Stub
+end
+
+def Stub(url, file_path)
```

Comment:
The capitalised method looks a bit unconventional, is there a strong reason for this?

Reply:
It is to save on typing, I used a method with the same name as the Module (See: https://stackoverflow.com/a/20330415/238230). Else it would be `Stub.create` or something. Let me know if you feel strongly against this approach.

Reply:
I guess that my question then is, why not just use a RSpec helper https://relishapp.com/rspec/rspec-core/docs/helper-methods/define-helper-methods-in-a-module

Reply:
Forgot to say, I don't have strong feelings, I just found it unconventional :)

Reply:
Thanks @arnau, I did not no about the RSpec Helper approach. I pushed a change to use that now.